doc/conf.py

```diff
@@ -26,8 +26,8 @@
 # The suffix of source filenames.
 source_suffix = '.rst'
 
-# The master toctree document.
-master_doc = 'index'
+# The root toctree document.
+root_doc = 'index'
```

Comment:
Changed in sphinx 4.0. See https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-root_doc